https://github.com/ddclient/ddclient/commit/a12398c315b9b909e57e87acf9fd3a15a0b3e213.patch
https://bugs.gentoo.org/936224

From: Richard Hansen <rhansen@rhansen.org>
Date: Thu, 19 Dec 2024 05:00:56 -0500
Subject: [PATCH] Makefile.am: Fix Automake portability warning

This silences:

    Makefile.am:20: warning: escaping \# comment markers is not portable
--- a/Makefile.am
+++ b/Makefile.am
@@ -16,19 +16,7 @@ EXTRA_DIST = \
 	sample-get-ip-from-fritzbox
 CLEANFILES =
 
-# Command that replaces substitution variables with their values.
-subst = sed \
-	-e 's|@PACKAGE_VERSION[@]|$(PACKAGE_VERSION)|g' \
-	-e '1 s|^\#\!.*perl$$|\#\!$(PERL)|g' \
-	-e 's|@localstatedir[@]|$(localstatedir)|g' \
-	-e 's|@runstatedir[@]|$(runstatedir)|g' \
-	-e 's|@sysconfdir[@]|$(sysconfdir)|g' \
-	-e 's|@CURL[@]|$(CURL)|g'
-
-# Files that will be generated by passing their *.in file through
-# $(subst).
 subst_files = ddclient ddclient.conf
-
 EXTRA_DIST += $(subst_files:=.in)
 CLEANFILES += $(subst_files)
 
@@ -36,7 +24,14 @@ $(subst_files): Makefile
 	rm -f '$@' '$@'.tmp
 	in='$@'.in; \
 		test -f "$${in}" || in='$(srcdir)/'$${in}; \
-		$(subst) "$${in}" >'$@'.tmp && \
+		sed \
+			-e 's|@PACKAGE_VERSION[@]|$(PACKAGE_VERSION)|g' \
+			-e '1 s|^#\!.*perl$$|#\!$(PERL)|g' \
+			-e 's|@localstatedir[@]|$(localstatedir)|g' \
+			-e 's|@runstatedir[@]|$(runstatedir)|g' \
+			-e 's|@sysconfdir[@]|$(sysconfdir)|g' \
+			-e 's|@CURL[@]|$(CURL)|g' \
+			"$${in}" >'$@'.tmp && \
 		{ ! test -x "$${in}" || chmod +x '$@'.tmp; }
 	mv '$@'.tmp '$@'
 
